.github/workflows/cpack.yml

```diff
@@ -180,13 +180,13 @@ jobs:
           if-no-files-found: error
 
       - name: Configure jFrog CLI
-        if: ${{ github.event.release.published && startsWith(github.ref, 'refs/tags/v') }}
+        if: ${{ !github.event.release.prerelease && startsWith(github.ref, 'refs/tags/v') }}
```

Comment:
It would be great if anyone can double check me on this event structure. I looked at the docs and [a recent event](https://github.com/openziti/ziti-tunnel-sdk-c/actions/runs/5894991944/job/15989783349) (select gear icon -> view raw logs)